### PR TITLE
Feature/reference aquality.selenium.core

### LINKED ISSUE #55: Add getName() and getLocator() into Form
Use case: In test solution, assert that form is present, use form's name/locator in assertion message

### LINKED ISSUE #60: add method like 'hasBrowserInstance' to BrowserManager

### LINKED ISSUE #66: Add findChildElements mefod for elements
Now it's impossible to check existing subElement without exception.
It has got workflow like get Xpath of the parent element, add Xpath of the child and search as new element on the page.

### LINKED ISSUE #69: It is impossible to use custom extended ElementFactory
If I want to create a class that extends 'ElementFactory' (e.g. I need to use method to find elements with wait), I cannot use this class in classes extend 'Form'.

### LINKED ISSUE #74: fix corrupted character encoding in '4.2. Custom elements' section of doc (both en/ru)
As of now, the `getCustomElement` method appears as "\&lt;T extends IElement\&gt; T getCustomElement". For it to display correctly, this fragment should be enclosed in reverse quotation marks instead of backslashes.

### PR DESCRIPTION
**Pull request by mialeska**
Reference aquality.selenium.core, aim to publish version 2.0.0 with support of DI (Guice).
closes #55, closes #60,  closes #66,  fixes #69, fixes #74

### CODE REVIEW

File: src/main/java/aquality/selenium/elements/ComboBox.java [hunk 1]
```diff
@@ -77,15 +76,16 @@ private void selectOptionThatContains(Function<WebElement, String> getValueFunc,
                 }
             }
             if (!isSelected){
-                throw new InvalidElementStateException(String.format(getLocManager().getValue("loc.combobox.impossible.to.select.contain.value.or.text"), value, getName()));
+                throw new InvalidElementStateException(String.format(getLocalizationManager().getLocalizedMessage(
```
Comment (knysh) [inline src/main/java/aquality/selenium/elements/ComboBox.java]: lets removed localized message here.

Comment (mialeska) [reply src/main/java/aquality/selenium/elements/ComboBox.java]: Let's do this in a separate issue #76

File: pom.xml [hunk 2]
```diff
@@ -67,6 +67,12 @@
     </developers>
 
     <dependencies>
+        <dependency>
```
[the remark this answers is not in the document]
Comment (mialeska) [reply pom.xml]: fixed

File: pom.xml [hunk 3]
```diff
@@ -67,6 +67,12 @@
...
+            <groupId>com.github.aquality-automation</groupId>
+            <artifactId>aquality-selenium-core</artifactId>
+            <version>0.0.1</version>
```
Comment (DmitryBogatko) [inline pom.xml]: change to 0.0.2 please

Comment (mialeska) [reply pom.xml]: fixed

**Pull request merged**